back-end/services/tvmaze.service.js

```diff
@@ -0,0 +1,13 @@
+const fetch = require('node-fetch');
+
+var TVMazeService = {
```

Comment:
Also out of curiousity, why not use let?

Reply:
Could also be a constant, since it doesn't get modified. In the beginning, I tried to keep it ES5 only, but looking at the project currently, it already is a mix between `var` and `const/let` maybe someone will go through it in the future and update them.